hammerdb/entrypoint

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Wouldn't this file make uid_entrypoint file redundant ? in that case can we remove one from the other ?

Reply:
I don't see this added to the dockerfile as well?